web-client/src/app/views/register/register.page.ts

```diff
@@ -149,4 +176,75 @@ export class RegisterPage implements OnDestroy {
   ngOnDestroy() {
     this.subscription$.unsubscribe();
   }
+
+  protected async notifyResult(
+    result: { xrplResult: TxResponse },
+    amount: AssetAmount,
+    receiverAddress: string
+  ): Promise<void> {
+    if ('xrplResult' in result) {
+      const { xrplResult: txResponse } = result;
+      const { succeeded, resultCode } = checkTxResponseSucceeded(txResponse);
+      if (succeeded) {
+        this.notifySuccess({
+          amount: `${formatAssetAmount(amount)} ${formatAssetSymbol(amount)}`,
+          address: receiverAddress,
+          txId: txResponse.id.toString(),
+          timestamp: new Date(),
+        });
+      } else {
+        await this.notifyXrplFailure({ resultCode });
+      }
+    }
+  }
+
+  protected notifySuccess({
+    amount,
+    address,
+    txId,
+    timestamp,
+    txUrlPrefix,
+  }: {
+    amount: string;
+    address: string;
+    txId: string;
+    timestamp: Date;
+    txUrlPrefix?: string;
+  }): void {
+    const txIdHtml = txUrlPrefix
+      ? `<a href="${txUrlPrefix}${txId}" target="_blank" >${txId}</a>`
+      : `${txId}`;
+    this.notification.swal.fire({
+      icon: 'success',
+      titleText: 'Wallet Created!',
+      text: 'Your wallet was successfully created.',
+      html: `<div >
```

Comment:
you need to choose 1, you can't have both text and html at the same time, [html will be used if both are provided](https://sweetalert2.github.io/#html)

Reply:
Going with html here due to formatting.